active/003-seq-feature.md

```diff
@@ -0,0 +1,180 @@
+---
+Feature name: sequence-positional-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt"]
+---
+
+# Summary
+
+Modify the `Sequence` class to incorporate the new pandas `IntervalIndex` into `positional_metadata`.
+
+# Motivation
+
+The current implementation of `positional_metadata` in the `Sequence` class takes > 1TB memory to read in a E. coli genome with 260K features. And the `pandas` sparse data frame can alleviate the mem usage, but it is still too memory hogging. The discussion is mainly in this [thread](https://github.com/biocore/scikit-bio/issues/1159).
+
+
+# Detailed design
+
+We propose to use `IntervalIndex` in `pandas` to store the interval features. It is still in active development and not merged into offical branch. There are some methods that don't aren't implemented yet(i.e. `IntervalIndex.is_non_overlapping_monotonic`). However, this is in high priority of `pandas` milestone and will eventually be merged.
+
+To run some of the code below, you will need the specific branch of `pandas` from github:
+
+```
+git clone git@github.com:shoyer/pandas.git
+cd pandas
+git pull origin IntervalIndex
+pip install -e .
+```
+
+## `interval_metadata` object structure
+
+1. We will have per position data frame (`positional_metadata`) and interval data frame (`interval_metadata`) to store the sequence metadata.
+
+2. The `interval_metadata` dataframe has following index or columns:
+
+   1) an `IntervalIndex` as its row index (named `INTERVAL`).
```

Comment:
I don't know if I am sold on the column names being all uppercase, but that is a minor detail.